### DISCUSSION TITLE
Github Pages not loading JavaScript files

### DISCUSSION THREAD

**Discussion by atul288** [community/community#23973] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Github,<br>
I want help from you I have uploaded my React project on Github and I follow all the instructions that are mention on React app website.<br>
And I uploaded it but when I run my app it will throw an error</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/e/7e1be2514bc2e41d729402c94110d949fee15ec2.png" data-download-href="/uploads/short-url/hZBTXxaKg7ra93m70DyWX61JfIS.png?dl=1" title="Screenshot (69)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/57065280/181080453-369834d9-ad65-4fa4-b0b9-1ed7a1603be7.png" alt="Screenshot (69)" data-base62-sha1="hZBTXxaKg7ra93m70DyWX61JfIS" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/e/7e1be2514bc2e41d729402c94110d949fee15ec2_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/e/7e1be2514bc2e41d729402c94110d949fee15ec2_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/e/7e1be2514bc2e41d729402c94110d949fee15ec2_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/e/7e1be2514bc2e41d729402c94110d949fee15ec2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (69)</span><span class="informations">1920×1080 153 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> .

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/atul288">@atul288</a>, welcome to the GitHub Support Community! Can you post a link to the repository you’re using to host your site?</p>

**Comment by atul288** [comment]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a>, thanks for your response here is my Repository URL.<br>
<a href="https://github.com/atul288/emojipedia-web-application.git" rel="noopener nofollow ugc">This is the Repository URL </a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for posting that! Looking at your site it seems like your asset URLs have an extra <code>/atul288</code> section that’s causing the links to be invalid:</p>
<pre><code class="lang-auto">&lt;script src="/atul288/emojipedia-web-application.git/static/js/2.6c736c09.chunk.js"&gt;&lt;/script&gt;
</code></pre>
<p>This is most likely caused by setting the <a href="https://github.com/atul288/emojipedia-web-application/blob/7bf14ba6f8767343fe420b64170f2bdc85d5af91/package.json#L2"><code>homepage</code> value inside your <code>package.json</code></a> incorrectly. It looks like you’ve set it to the URL of your GitHub repository:</p>
<pre><code class="lang-auto">"homepage": "https://github.com/atul288/emojipedia-web-application.git",
</code></pre>
<p>You’ll need to set this to the URL of your site instead:</p>
<pre><code class="lang-auto">"homepage": "https://atul288.github.io/emojipedia-web-application/",
</code></pre>

**Reply by hikouki-gumo** [reply]
Hi, how the homepage is used then? I don't see it in build folder, how github pages know homepage?

**Comment by atul288** [comment]
<p>Thanks for your suggestions.<br>
I’ll do this as well and get back to you soon.</p>

**Comment by atul288** [comment]
<p><a class="mention" href="/thomasshaped">@thomasshaped</a>  ,<br>
Thanks to you now it’s working fine exactly what is wanted.</p>

**Comment by rahulkumar0509** [comment]
Gthub pages is not uploading any js files in deployed application.
https://github.com/rahulkumar0509/food-blog 
Can you please what could be wrong. its dummy project and I followed all the steps to deploy using github pages but its not working.

**Comment by goody09083729531** [comment]
Hi, Github 
Please I've hosted my Site using Github Pages but my Javascript Code does not seem to be Previewed and thereby eliminating some of the features of that site.
Pls what can I do

**Reply by chrisbaks** [reply]
you need to post  link to the page repository so others can view your code and offer solutions

**Comment by GerardPaul** [comment]
I have the same issue in my Angular app. I am at a loss. This is my repo. https://github.com/GerardPaul/TheGPWorxUI
![image](https://github.com/community/community/assets/4158008/ac06f0c2-fef0-49be-85d6-d8e5859d61ff)
